4-Express/client/scripts/communication_manager.js

```diff
@@ -0,0 +1,38 @@
+const axios = require("axios").default;
+const SERVER_URL = "http://localhost:3000";
+
+function addOrUpdateItem(id, item) {
+  const urlWithId = `${SERVER_URL}/${id}`;
+
+  axios.post(urlWithId, item).then(
+    (response) => {
+    },
+    (error) => {
+      console.log(error);
+    }
```

Comment:
this amazing error handling technique is repeated several times. export to a function